demo/dialog-basic-demos.html

```diff
@@ -44,23 +38,18 @@ <h3>Styling the Content</h3>
             border: .25em solid #000;
           }
         </style>
-        <dom-bind id="styled-dialog-example">
+        <vaadin-dialog>
           <template>
-            <vaadin-dialog id="styleddialog">
-              <template>
-                <div class="dialog-message">I am styled from the parent scope and can be closed by pressing the Esc key,
-                or by a mouse click anywhere outside the dialog area</div>
-              </template>
-            </vaadin-dialog>
-            <vaadin-button on-click="showStyledDialog">Show dialog</vaadin-button>
+            <div class="dialog-message">I am styled from the parent scope and can be closed by pressing the Esc key,
+            or by a mouse click anywhere outside the dialog area</div>
           </template>
-        </dom-bind>
+        </vaadin-dialog>
+        <vaadin-button on-click="showStyledDialog">Show dialog</vaadin-button>
```

Comment:
The `on-click="showStyledDialog"` should also be removed.

Reply:
Removed